src/borg/remote.py

```diff
@@ -370,6 +370,8 @@ def inject_exception(self, kind):
             raise Repository.CheckNeeded(s1)
         elif kind == 'IntegrityError':
             raise IntegrityError(s1)
+        elif kind == 'AtticRepository':
+            raise Repository.AtticRepository(s1)
```

Comment:
I had the impression that we want to be able to test any exception with this function but that does not seem to be the case. Will remove this hunk.